src/core/stream_recv.c

```diff
@@ -999,6 +999,13 @@ QuicStreamReceiveComplete(
         //
         QuicStreamTryCompleteShutdown(Stream);
 
+        //
+        // Flush the connection-wide MAX_DATA.
+        //
+        QuicSendSetSendFlag(
```

Comment:
When is a `MAX_DATA` sent otherwise? when stream drains that are not `FIN` happen?